docs/admin/managing_clipboard.rst

```diff
@@ -0,0 +1,53 @@
+
+1. It is dangerous to copy untrusted content *into* a secure environment. What looks like plain text may contain character sequences that exploit security vulnerabilities in the target environment.
+2. The four-step process described above can be difficult to follow, and it is easy to make an operational mistake, such as pasting the wrong content into a message to a source.
+3. SecureDrop Workstation aims to minimize the `attack surface <https://en.wikipedia.org/wiki/Attack_surface>`__ adversaries can target to exfiltrate information. For example, if the clipboard implementation in Qubes OS itself contained a security vulnerability, an adversary could attempt to exploit that vulnerability to shuttle a message sent by a source from the networkless ``sd-app`` VM to a networked VM.
+
+With these considerations in mind, there are use cases where clipboard access may be an important part of your regular use of SecureDrop Workstation. For example:
+
+- You may want to copy passwords from a password manager to the SecureDrop Client;
+- You may want to copy a message you received via SecureDrop into a secure messaging app like Signal, to share it with another journalist.
+
+To support these use cases, SecureDrop Workstation allows you to grant granular access to the ``sd-app`` clipboard (via the cross-VM clipboard) to selected VMs.
+
+Configuring clipboard access to ``sd-app``
+------------------------------------------
+
+The process for setting up a password manager and permitting the one-directional copying of passwords to the SecureDrop Client is :ref:`outlined in the installation docs <Password Management Section>`. In general, clipboard access to SecureDrop Workstation VMs is governed by *tags* that can be applied in ``dom0`` to selected VMs:
+
+- the tag ``sd-send-app-clipboard`` can be used to tag a VM that should be able to send its clipboard contents *to* ``sd-app`` via the cross-VM clipboard;
+- the tag  ``sd-receive-app-clipboard`` can be used to tag a VM that should be able to receive is clipboard contents *from* ``sd-app`` via the cross-VM clipboard.
+
+You can configure these tags for a given VM by opening a terminal in ``dom0`` and issuing a command like the following:
+
+.. code-block:: sh
+
+  qvm-tags <VM name> add <tag name>
+
+So, for example, if you had a VM for the Signal messenger called ``signal`` and you wanted to copy messages *from* the SecureDrop Client *to* Signal, you would issue the following command:
+
+.. code-block:: sh
+
+  qvm-tags signal add sd-receive-app-clipboard
+
+The syntax for revoking a tag is as follows:
+
+.. code-block:: sh
+
+  qvm-tags <VM name> del <tag name>
+
+Changes to tags take effect immediately, and any VM can have multiple tags.
+
+.. important::
```

Comment:
I would suggest we move this at the top of this page

Reply:
Moved up, but not before the terminology is introduced.